source/wp-content/themes/wporg-parent-2021/sass/base/_elements.scss

```diff
@@ -28,6 +28,21 @@ mark.has-inline-color {
 	}
 }
 
+// For links that navigate users to different site
+a:not([href*="wordpress.org"])::after {
+	content: "";
+	display: inline-flex;
+	background-color: var(--wp--preset--color--charcoal-4);
+	width: 1em;
+	height: 1em;
+	margin-left: 0.1em;
+	vertical-align: middle;
+	-webkit-mask-image: url(../images/external-link-icon.svg);
+	mask-image: url(../images/external-link-icon.svg);
+	-webkit-mask-size: cover;
```

Comment:
The build process here has autoprefixer, so that should handle adding in the `-webkit-` properties.